webrtc.html

```diff
@@ -8905,20 +8918,26 @@ <h4 id="sec.idp-loginneeded">User Login Procedure</h4>
         is unable to verify that a user is authenticated. This might be due to
         the IdP not having the necessary authentication information available
         to it (such as cookies).</p>
+
         <p>Rejecting the promise returned by <code><a data-for=
         "RTCIdentityProvider">generateAssertion</a></code> will cause the error
-        to propagate to the application. Login errors are indicated by
-        rejecting the promise with an object that has a <code>name</code>
-        attribute set to "IdpLoginError".</p>
-        <p>If the rejection object also contains a <code>loginUrl</code>
-        attribute, this value will be passed to the application in the
-        <code>idpLoginUrl</code> attribute. This URL might link to page where a
-        user can enter their (IdP) username and password, or otherwise provide
-        any information the IdP needs to authorize a assertion request.</p>
+        to propagate to the application. Login errors are indicated by rejecting
+        the promise with an <code>RTCError</code> with errorDetail set to
+        "IdPNeedLogin" and the <code>idpLoginUrl</code> attribute set to the URL
+        where the user can log on.  This URL might link to page where a user can
+        enter their (IdP) username and password, or otherwise provide any
+        information the IdP needs to authorize a assertion request. </p>
+
+        <p>The value of the <code>idpLoginUrl</code> attribute contained in the
+        rejection will also be passed to the application in the
+        <code>idpLoginUrl</code> attribute of the
+        <code>RTCPeerConnection</code>.</p>
```

Comment:
On Nov 9 WG call, decided to remove the idpLoginUrl from this PR and move to a separate bug that can be considered on it's own ( And waits till we decide if we do multiple identity providers or not first)